code/modules/power/lighting.dm

```diff
@@ -888,7 +888,7 @@
 	. = ..()
 	AddComponent(/datum/component/caltrop, force)
 
-/obj/item/light/proc/on_entered(datum/source, atom/movable/L)
+/obj/item/light/proc/on_entered(datum/source, mob/living/L)
```

Comment:
this isn't solution
check at the beggining if atom movable isn't mob living and return from there, don't mess with arguments

Reply:
just did it, thanks for the heads up